ptx/sec_limit_continuity.ptx

```diff
@@ -118,13 +118,15 @@
             \end{tikzpicture}
           </latex-image>
           <description>
-            Shows the domain 0 to 3.5 and range 0 to 2. For <m>0 \geq x &lt; 1</m> 
-            the graph has a downward curve to it, for <m>1 &gt; x \leq 2</m> the graph
-            is a straight line, parralel with the x-axis, and for <m>2 \geq x \leq 3</m>.
-            The graph is undefined at <m>x = 1</m>. 
+            <p>
+              Shows the domain 0 to 3.5 and range 0 to 2. For <m>0 \geq x &lt; 1</m> 
+              the graph has a downward curve to it, for <m>1 &gt; x \leq 2</m> the graph
+              is a straight line, parralel with the x-axis, and for <m>2 \geq x \leq 3</m>.
+              The graph is undefined at <m>x = 1</m>. 
+            </p>
           </description>
           <shortdescription>
-
+            A graph of f in <xref ref="ex_contint1">Example</xref>
```

Comment:
cross references will not work in the short description. You need to use something like "the current example" or "the above example"